app/src/main/java/com/cris/nvh/framgiaproject/screen/mymusic/MyMusicFragment.java

```diff
@@ -80,14 +81,17 @@ public void onClick(View view) {
 		}
 	}
 
-	public static Intent getPlayActivityIntent(Context context, Track track) {
+	public static Intent getPlayActivityIntent(Context context, ArrayList<Track> tracks, int index) {
```

Comment:
```suggestion
	public static Intent getPlayActivityIntent(Context context, List<Track> tracks, int index) {
```
Em nên thống nhất là List<> thay cho ArraList<>. sd tính đa hình trong Java
Sửa những nơi khác luôn nhé.